### ISSUE TITLE
Consider scrapping majority of current FAQ

### ISSUE THREAD

**Issue by yminsky** [ocaml/v2.ocaml.org#236]
This is perhaps a contentious suggestions, but the current FAQ seems pretty messy.  It has some out of date suggestions (Num is basically deprecated, and people should probably use zarith instead), some that seem out of place in a FAQ ("how to define an enumerated type" seems like a better topic for a tour of the language, rather than an FAQ), lots of little grammatical mistakes, ("How to define multidimensional arrays" should probably be something like "How do you define a multideminsional array?", or "Types definitions" should be "Type definitions".

All told, the FAQ seems long and not of very high quality.  I'd recommend starting from an empty page and pulling in FAQs from the list that seem good, rather than the other way around.

**Comment by avsm** [comment]
I agree with this.  I'm happy to help draft questions when I get back to Cambridge next week if there's a general desire to do this...

**Comment by agarwal** [comment]
The FAQ section of the site has never been good. It was an experimental section that we were hoping would get better through contributions, which never happened. I'm happy to scrap it completely.

**Comment by avsm** [comment]
Got it; I'll chat with @samoht and @altgr about an OPAM section there too -- there are a common emerging set of 'workflow questions' emerging that would be perfect here.

**Comment by agarwal** [comment]
We could create a separate _faq_ branch and start adding there. If it gets into shape, we can merge it in. In the mean time, there won't be a poorly maintained section live.

**Comment by avsm** [comment]
Does a "why ocaml" page fit into the FAQ?  There are a number of articles such as:
http://roscidus.com/blog/blog/2013/06/09/choosing-a-python-replacement-for-0install/

which would be good to link to from one place.  It's certainly a frequently asked question, but may be better as a dedicated page.

**Comment by agarwal** [comment]
I think of FAQs as questions that can be answered in at most a paragraph or
two. "Why OCaml" sounds like a whole page to me. We could have a long essay
and also link to external articles.

On Mon, Nov 11, 2013 at 8:58 PM, Anil Madhavapeddy <notifications@github.com

**Comment by yminsky** [comment]
For what it's worth, RWO has a "why ocaml" essay at the beginning:

https://realworldocaml.org/v1/en/html/prologue.html
